src/190B20.c

```diff
@@ -1897,15 +2771,48 @@ void func_80266D6C(ActorPart* part, s32 arg1) {
     }
 }
 
-INCLUDE_ASM(s32, "190B20", func_80266DAC);
+void func_80266DAC(Actor* actor, s32 arg1) {
+    ActorPart* partIt = &actor->partsTable[0];
+
+    while (partIt != NULL) {
+        if (!(partIt->flags & ACTOR_PART_FLAG_INVISIBLE) &&
+           (partIt->idleAnimations != NULL) &&
+           !(partIt->flags & ACTOR_PART_FLAG_2))
+        {
+            func_80266D6C(partIt, arg1);
+        }
+        partIt = partIt->nextPart;
+    }
+}
 
 void func_80266E14(ActorPart* part) {
     if (part->idleAnimations != NULL && !(part->flags & ACTOR_PART_FLAG_2)) {
         part->decorationTable->unk_6C0 = 0;
     }
 }
 
-INCLUDE_ASM(s32, "190B20", func_80266E40);
+// TODO: improve match
+void func_80266E40(Actor* actor) {
+    ActorPart* partIt = actor->partsTable;
+    s8 e = 0xE;
+    s8 f = 0xF;
```

Comment:
Are these required to be hex for it to match?

Reply:
No, but they're enum values so it makes more sense to me (also is more obvious that it's wrong for fixing later)